### PR TITLE
refactor(backend): rename `evenProtocolType` to `eventType`

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

rename `evenProtocolType` to `eventType`

### Issues

- based on #2775 
- contributes to #2722

### Todo

- [X] None

### CODE REVIEW

File: backend/src/graphql/resolver/UserResolver.ts [hunk 1]
```diff
@@ -283,14 +289,16 @@ export class UserResolver {
...
+          eventRegisterRedeem.involvedContribution = { id: contributionLink.id } as DbContribution
         }
       } else {
         const transactionLink = await DbTransactionLink.findOne({ code: redeemCode })
         logger.info('redeemCode found transactionLink=' + transactionLink)
         if (transactionLink) {
           dbUser.referrerId = transactionLink.userId
-          eventRegisterRedeem.transactionId = transactionLink.id
+          // TODO this is so wrong
+          eventRegisterRedeem.involvedTransaction = { id: transactionLink.id } as DbTransaction
```
Comment (clauspeterhuebner) [inline backend/src/graphql/resolver/UserResolver.ts]: if it is "so wrong", why did you do it or why not refactor?

**Review by clauspeterhuebner — APPROVED** [review]
some comments, but i approve it, because of previous discussions and good done job...

File: backend/src/graphql/resolver/UserResolver.ts [hunk 2]
```diff
@@ -283,14 +289,16 @@ export class UserResolver {
         logger.info('redeemCode found contributionLink=' + contributionLink)
         if (contributionLink) {
           dbUser.contributionLinkId = contributionLink.id
-          eventRegisterRedeem.contributionId = contributionLink.id
+          // TODO this is so wrong
+          eventRegisterRedeem.involvedContribution = { id: contributionLink.id } as DbContribution
```
[the remark this answers is not in the document]
Comment (ulfgebhardt) [reply backend/src/graphql/resolver/UserResolver.ts]: this is fixed in a later commit

File: backend/src/graphql/resolver/UserResolver.ts (hunk 1 above)
Comment (ulfgebhardt) [reply backend/src/graphql/resolver/UserResolver.ts]: this is fixed in a later commit

**Pull request merged**